script.js

```diff
@@ -19,14 +19,140 @@ function hamburgerAction() {
+    workLanguages: ['css', 'html', 'bootstrap', 'Ruby'],
+    liveLink: 'https://kazim110.github.io/',
+    projLink: 'https://github.com/kazim110/My-Portfolio-Project',
+  },
+  {
+    workId: '2',
+    workTitle: 'Multi-Post Stories',
+    workImg: 'snapshoot-portfolio1.png',
+    workDesc: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    workLanguages: ['css', 'html', 'bootstrap', 'Ruby'],
+    liveLink: 'https://kazim110.github.io/',
+    projLink: 'https://github.com/kazim110/My-Portfolio-Project',
+  },
+  {
+    workId: '3',
+    workTitle: 'Multi-Post Stories',
+    workImg: 'snapshoot-portfolio2.png',
+    workDesc: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    workLanguages: ['css', 'html', 'bootstrap', 'Ruby'],
+    liveLink: 'https://kazim110.github.io/',
+    projLink: 'https://github.com/kazim110/My-Portfolio-Project',
+  },
+  {
+    workId: '4',
+    workTitle: 'Multi-Post Stories',
+    workImg: 'snapshoot-portfolio3.png',
+    workDesc: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    workLanguages: ['css', 'html', 'bootstrap', 'Ruby'],
+    liveLink: 'https://kazim110.github.io/',
+    projLink: 'https://github.com/kazim110/My-Portfolio-Project',
+  },
+];
+
+const cards = document.querySelector('.cards');
+
+cards.innerHTML = '';
+
+for (let j = 0; j < workDetails.length; j += 1) {
+  let techLang = '';
+  workDetails[j].workLanguages.forEach((workLanguages) => {
+    techLang = `${techLang}<li>${workLanguages}</li>`;
+  });
+
+  cards.innerHTML += `<div class="card">
+<img class="work-img" src="./img/${workDetails[j].workImg}" alt="Avatar">
+<div class="work-description">
+    <div class="work-title">
+        <h4 class="work-title-main">${workDetails[j].workTitle}</h4>
+        <ul class="work-info">
+            <li class="client">
+                <p>
+                    Canopy
+                </p>
+            </li>
+            <li class="role">
+                <p>
+                    Back End Dev
+                </p>
+            </li>
+            <li class="year">
+                <p>
+                    2015
+                </p>
+            </li>
+        </ul>
+    </div>
+    <div class="work-content-text">
+        <p>
+          ${workDetails[j].workDesc} 
+        </p>
+    </div>
+    <ul class="work-tech-used">
+      ${techLang}
+    </ul>
+    <div class="action-div">
+        <button class="secondary-btn  work-modal-${j}"><p>See Project</p></button>
```

Comment:
- [ ] Good job on the design you made, it looks great 👏 on thing I would suggest is to add some spacing between the two blocks as illustrated below 👇 
![image](https://user-images.githubusercontent.com/90222110/204038043-90c373e9-c089-4e4f-9264-f6b7e082b12f.png)


As you can see on the design, there is a considerable space
![image](https://user-images.githubusercontent.com/90222110/204038113-5978e670-fa91-41f5-b30f-21c81a2a8165.png)